.eslintignore

```diff
@@ -1 +1,3 @@
+# SPDX-FileCopyrightText: 2023 Nextcloud GmbH and Nextcloud contributors
+# SPDX-License-Identifier: CC0-1.0
```

Comment:
Why it cannot be set in reuse like for other config files?

Reply:
We can, but it's "better" to be put in the file directly, especially when it's not shipped.